src/Avalonia.Controls/Platform/InternalPlatformThreadingInterface.cs

```diff
@@ -1,10 +1,7 @@
 ﻿using System;
```

Comment:
I believe this class should have internal-like attributes as we have now in other places:
@kekekeks, what do you think?
```
    [Obsolete("Internal API, could be removed in future versions, you've been warned")]
    [System.ComponentModel.Browsable(false)]
    [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
```